packages/messaging-api-line/README.md

```diff
@@ -143,70 +146,71 @@ client.reply(REPLY_TOKEN, 'Hello!');
+| image.previewImageUrl    | `String` | Preview image URL.                 |
 
 Example:
 
 ```js
-client.replyImage(
-  REPLY_TOKEN,
-  'https://example.com/original.jpg',
-  'https://example.com/preview.jpg'
-);
+client.replyImage(REPLY_TOKEN, {
+  originalContentUrl: 'https://example.com/original.jpg',
+  previewImageUrl: 'https://example.com/preview.jpg',
+});
 ```
 
 <br />
 
-## `replyVideo(token, videoUrl, previewImageUrl)` - [Official Docs](https://developers.line.me/en/docs/messaging-api/reference/#video-message)
+## `replyVideo(token, video)` - [Official Docs](https://developers.line.me/en/docs/messaging-api/reference/#video-message)
 
 Responds video message using specified reply token.
 
 <img src="https://developers.line.me/media/messaging-api/messages/video-a1bc08a4.png" width="250px" />
 
-| Param           | Type     | Description                        |
-| --------------- | -------- | ---------------------------------- |
-| token           | `String` | `replyToken` received via webhook. |
-| videoUrl        | `String` | URL of video file.                 |
-| previewImageUrl | `String` | URL of preview image.              |
+| Param                    | Type     | Description                        |
+| ------------------------ | -------- | ---------------------------------- |
+| token                    | `String` | `replyToken` received via webhook. |
+| video.originalContentUrl | `String` | URL of video file.                 |
+| video.previewImageUrl    | `String` | URL of preview image.              |
 
 Example:
 
 ```js
-client.replyVideo(
-  REPLY_TOKEN,
-  'https://example.com/original.mp4',
-  'https://example.com/preview.jpg'
-);
+client.replyVideo(REPLY_TOKEN, {
+  originalContentUrl: 'https://example.com/original.mp4',
+  previewImageUrl: 'https://example.com/preview.jpg',
+});
 ```
 
 <br />
 
-## `replyAudio(token, audioUrl, duration)` - [Official Docs](https://developers.line.me/en/docs/messaging-api/reference/#audio-message)
+## `replyAudio(token, audio)` - [Official Docs](https://developers.line.me/en/docs/messaging-api/reference/#audio-message)
 
 Responds audio message using specified reply token.
 
 <img src="https://developers.line.me/media/messaging-api/messages/audio-6290d91b.png" width="250px" />
 
-| Param    | Type     | Description                        |
-| -------- | -------- | ---------------------------------- |
-| token    | `String` | `replyToken` received via webhook. |
-| audioUrl | `String` | URL of audio file.                 |
-| duration | `Number` | Length of audio file.              |
+| Param                    | Type     | Description                        |
+| ------------------------ | -------- | ---------------------------------- |
+| token                    | `String` | `replyToken` received via webhook. |
+| audio.originalContentUrl | `String` | URL of audio file.                 |
+| audio.duration           | `Number` | Length of audio file.              |
```

Comment:
We can add unit of duration here (seconds or ms).

Reply:
According to the official doc, it should be `milliseconds`
> Length of audio file (milliseconds)